evoseal/integration/seal/data_loaders/loaders.py

```
@@ -38,13 +38,13 @@ def from_file(cls, file_path: Union[str, Path], model: Type[T], **kwargs: Any) -
         if not file_path.exists():
             raise FileNotFoundError(f"File not found: {file_path}")
 
-        format = file_path.suffix.lstrip('.').lower()
+        format = file_path.suffix.lstrip(".").lower()
         try:
             format_enum = DataFormat(format)
         except ValueError:
             raise ValueError(f"Unsupported file format: {format}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)